lib/esp.rb

```diff
@@ -1,26 +1,40 @@
 module ESP
   # Manually set the access_key_id you created from https://esp.evident.io/settings/api_keys.
   #
-  # You can optionally set the `ESP_ACCESS_KEY_ID` environment variable.
+  # You can optionally set the +ESP_ACCESS_KEY_ID+ environment variable.
+  #
+  # @param access_key_id [String] Your access key ID.
+  # @return [void]
   def self.access_key_id=(access_key_id)
     @access_key_id               = access_key_id
     ESP::Resource.hmac_access_id = access_key_id
   end
 
-  # Reads the `ESP_ACCESS_KEY_ID` environment variable if ::access_key_id was not set manually.
+  # Reads the +ESP_ACCESS_KEY_ID+ environment variable if +::access_key_id+ was not set manually.
```

Comment:
Everywhere there is a method preceded by `::`, as in `::access_key_id`, it's supposed to be a link to the documentation for that method.  It looks like the yard syntax for that would be `{#my_instance_method}` or `{.class_method}`